src/schema/v2/artworkQuiz/artworkQuiz.ts

```diff
@@ -0,0 +1,121 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import {
+  ArtworkConnectionInterface,
+  ArtworkEdgeInterface,
+  ArtworkType,
+} from "schema/v2/artwork"
+import {
+  connectionWithCursorInfo,
+  createPageCursors,
+} from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+
+const artQuizConnectionDefinition = connectionWithCursorInfo({
+  name: "QuizArtworks",
+  nodeType: ArtworkType,
+  edgeInterfaces: [ArtworkEdgeInterface],
+  connectionInterfaces: [ArtworkConnectionInterface],
+})
+
+const QUIZ_GENES: string[] = [
+  "trove",
+  "artists-on-the-rise",
+  "top-auction-lots",
+  "finds-under-1000",
+  "street-art-now-1",
+  "trending-this-week",
+  "iconic-prints",
+]
+
+const COLLECTION_ID = "contemporary-now"
+
+export const artQuizConnection: GraphQLFieldConfig<void, ResolverContext> = {
+  type: artQuizConnectionDefinition.connectionType,
```

Comment:
If this field is returning a connection of artworks (ie- a set of edges with artworks as their node), unless there's something different about the shape (of the edge), it should likely re-use the existing `ArtworkConnection` type (rather than defining a new type called `QuizArtworksConnection` which is just an artwork connection).